scripts/generate-badges.ps1

```diff
@@ -20,13 +21,6 @@ function Get-AzureLine($branchName, $jobNames, [switch]$integration = $false) {
     foreach ($jobName in $jobNames) {
 
         $configName = ""
-        $i = $jobName.IndexOf('#')
```

Comment:
I deleted this because I removed the use, unless I should be using this for some of the jobs we do today and I don't realize it?